UKHO.ADM/internal/ConvertTo-GroupObjects.ps1

```diff
@@ -20,6 +20,16 @@ function ConvertTo-GroupObjects {
                 $ggGroup = [ADGroup]::new($Settings.GroupPrefix, $Settings.Environment, $OU.Name, $GroupConfig.Name, "GG", $OU.DistinguishedName, $OU.Domain)
                 $dlGroup.ADGroupMembers += $ggGroup
 
+                # Add Other Groups
+                if ($null -ne $GroupConfig.Groups -and $GroupConfig.Groups.Count -gt 0) {
+                    $GroupConfig.Groups | ForEach-Object {
+                        $groupGroup = [ADGroup]::new($_.DistinguishedName, $OU.Domain)
+                        if (Check-GroupExists $groupGroup) {
+                            $dlGroup.ADGroupMembers += $groupGroup
```

Comment:
Same question as below about whether this adds to the correct group. Should this be the GG instead of the DL?

Reply:
this makes it the same code, which is even more amazing

Reply:
agreed :100: